json-resume-builder/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
I think it all looks pretty good. App imports all the sections and each section builds the appropriate HTML. I might come back to this project when I re-do mine using React.

Reply:
100% agree